<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nostaRahaa</class>
 <widget class="QDialog" name="nostaRahaa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn60">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn200">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn500">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMuu">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Muu summa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTakaisin">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOtaRahat">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/rahat.png);
font: 10pt &quot;Lucida Sans&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>OTA RAHAT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMuuSumma">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMuuSumma">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikko">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft Sans Serif&quot;;
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>	Paljonkos aattelit nostaa?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
